examples/dgfip_c/ml_primitif/stubs.c

```diff
@@ -614,19 +632,98 @@ static void init_var_dict(void)
   sort_index();
 }
 
-void efface_var(T_irdata *irdata, T_var_irdata desc)
+/*
+typedef struct idx_entry_t {
+  short start;
+  short end;
```

Comment:
Why this commented code here?

Reply:
It's just experimental code for a poor man's pseudo-hashtable to improve variable lookup times (actually doesn't really change anything in terms of performance). I don't know yet if i'll keep that or not, hence the comment.